src/elm/Liquidate.elm

```diff
@@ -339,31 +337,7 @@ view userLanguage currentTimeZone maybeConfig maybeNetwork account compoundState
 
 
 loadPresidioBorrows : Dict String String -> Network -> Cmd Msg
-loadPresidioBorrows apiBaseUrlMap network =
-    let
-        maybeAccountsUrl =
-            { addresses = []
-            , min_borrow_value_in_eth = Just minBorrowEthAmount
-            , max_health = Just maxHealthAmount
-            , block_number = 0
-            , page_size = 100
-            , page_number = 1
-            }
-                |> CompoundApi.Presidio.Accounts.Urls.accountsRequestUrl apiBaseUrlMap network
-
-        accountsRequestCmd =
-            case maybeAccountsUrl of
-                Just accountsUrl ->
-                    let
-                        accountsRequestHttpGet =
-                            Http.get accountsUrl accountsResponseDecoder
-                    in
-                    Http.send (ForSelf << PresidioAccountsResponse) accountsRequestHttpGet
-
-                _ ->
-                    Cmd.none
-    in
-    accountsRequestCmd
+loadPresidioBorrows apiBaseUrlMap network = Cmd.none
```

Comment:
Could could wipe out this function entirely also since it's only returning Cmd.none which is basically a NO-OP.